TriforkSwiftExtensions/TriforkSwiftExtensions/UIView/UIViewExtension.swift

```diff
@@ -113,6 +119,18 @@ public extension UIView {
     func findFirstResponder() -> UIView? {
         return findFirstResponder(inView: self)
     }
+
+    /// Adds `view` as subview to `self` and sets constraints for all edges with specified inset.
```

Comment:
Ehm.. Maybe with a default intset of `.zero`

Reply:
I thought of that, but if we do so it will be the same as the default `addSubview`. So we should give it another name if we want to do that - what do you think?
Maybe `addSubviewWithEdgeConstraints` ?

Reply:
Point taken!